cinema/views.py

```diff
@@ -48,6 +83,20 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+
+        movie = self.request.query_params.get("movie")
+        date = self.request.query_params.get("date")
+
+        if movie:
+            queryset = MovieSession.objects.filter(movie__id=int(movie))
```

Comment:
You shouldn't replace queryset, you should use queryset object.

Reply:
Not fixed